apps.toml

```
@@ -4020,6 +4020,11 @@ state = "working"
 subtags = [ "monitoring" ]
 url = "https://github.com/YunoHost-Apps/x-prober_ynh"
 
+[xbackbone]
+category = "small_utilities"
```

Or should it be `synchronization` ? Not sure to understand the app, it seems similar to stuff like Bozon, Droppy, FilePizza ?

```suggestion
category = "synchronization"
potential_alternative_to = [ "WeTransfer" ]
subtags = [ "files" ]
```